core/src/main/java/com/novoda/noplayer/internal/exoplayer/AdvertPlaybackState.java

```diff
@@ -0,0 +1,48 @@
+package com.novoda.noplayer.internal.exoplayer;
+
+import com.google.android.exoplayer2.source.ads.AdPlaybackState;
+import com.novoda.noplayer.Advert;
+import com.novoda.noplayer.AdvertBreak;
+
+import java.util.Collections;
+import java.util.List;
+
+public final class AdvertPlaybackState {
+
+    public static AdPlaybackState from(List<AdvertBreak> advertBreaks) {
+        Collections.sort(advertBreaks, new AdvertBreakStartTimeComparer());
+
+        long[] advertOffsets = advertBreakOffset(advertBreaks);
```

Comment:
The following is us creating the convoluted `AdPlaybackState` required by Exoplayer from the simple models we take in